org.gnome.Evolution.json

```diff
@@ -0,0 +1,374 @@
+
+
+    {
+        "app-id": "org.gnome.Evolution",
+        "runtime": "org.gnome.Platform",
+        "runtime-version": "3.28",
+        "sdk": "org.gnome.Sdk",
+        "command": "evolution",
+        "rename-icon": "evolution",
+        "copy-icon": true,
+        "build-options" : {
+            "cflags": "-O2 -g -Wno-deprecated-declarations",
+            "cxxflags": "-O2 -g -Wno-deprecated-declarations"
+        },
+        "cleanup": [
+            "/include",
+            "/lib/pkgconfig",
+            "/share/pkgconfig",
+            "/share/aclocal",
+            "*.la",
+            "*.a"
+        ],
+        "finish-args": [
+            "--share=ipc",
+            "--share=network",
+            "--socket=x11",
+            "--socket=wayland",
+            "--socket=pulseaudio",
+            "--talk-name=org.gnome.OnlineAccounts",
+            "--talk-name=org.freedesktop.Notifications",
+            "--talk-name=org.freedesktop.portal.Desktop"
+        ],
+        "modules": [
+            {
+                "name": "libical",
+                "cleanup": [ "/lib/cmake" ],
+                "buildsystem": "cmake-ninja",
+                "config-opts": [
+                    "-DCMAKE_INSTALL_LIBDIR:PATH=/app/lib",
```

Comment:
Shouldn't be needed.

Reply:
I'm not able to build Evolution after removing this module.